Library/Formula/shml.rb

```diff
@@ -0,0 +1,16 @@
+class Shml < Formula
+  desc "Style Framework for The Terminal"
+  homepage "http://shml.xyz"
+  url "https://github.com/MaxCDN/shml/archive/1.0.4.tar.gz"
+  sha256 "583fefbbf2adf45d989b052696be0f20f695903878558a4d2e9f74255b91ed51"
+
+  def install
+    bin.install "shml.sh"
```

Comment:
@DomT4 After some mucking around locally, this is what I'm looking for (assuming CI finally passes): https://github.com/jmervine/homebrew/commit/8baaa57961ff8b029d5cdae6c9236d40a49f9b62. 

I want both `shml.sh` and `shml` to be executable and to be able to be sourced in a subsequent bash script. 

Thanks again for your help.